pkg/api/defaults-kubelet.go

```diff
@@ -67,8 +67,8 @@ func (cs *ContainerService) setKubeletConfig() {
 		"--image-pull-progress-deadline":    "30m",
 	}
 
-	// AKS overrides
-	if cs.Properties.IsHostedMasterProfile() {
+	// Set --non-masquerade-cidr if ip-masq-agent is disabled on AKS
+	if cs.Properties.IsHostedMasterIPMasqAgentDisabled() {
```

Comment:
let's extend the unit tests to include this case

Reply:
Sure, will do.